opentelemetry/chart/templates/_helpers.tpl

```diff
@@ -7,8 +7,10 @@ Generic plugin name
 
 {{/* Generate plugin specific labels */}}
 {{- define "plugin.labels" -}}
-plugindefinition: opentelemetry 
-plugin: {{ $.Release.Name }}
+plugindefinition: opentelemetry
+{{- with $.Values.openTelemetry.prometheus.additionalLabels }}
+{{ tpl (toYaml . ) $ }}
+{{- end }}
```

Comment:
Just realized that all resources get the additional label. But that only needs to be added to the `ServiceMonitor`, `PodMonitor`, `PrometheusRules`, etc. - e.g. [here](https://github.com/cloudoperators/greenhouse-extensions/blob/638099cc7853b3940d4e7443a5931a3fcddbe240/opentelemetry/chart/templates/smon.yaml#L7-L16)
```yaml
kind: ServiceMonitor
metadata:
  annotations:
  name: opentelemetry-collector-metrics
  labels:
    {{- include "plugin.labels" . | nindent 4 }}
    {{- with .Values.openTelemetry.prometheus.additionalLabels }}
    {{ tpl (toYaml . ) $ }}
    {{- end }}
    {{- if .Values.customLabels }}
    {{ toYaml .Values.customLabels | nindent 4 }}
    {{- end }}
spec:
```

```suggestion
```